docs/api/migration-guides/local-simulators.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Migrate to local simulators
+description: Migrate from using the IBM Quantum cloud simulators (such as ibmq_qasm_simulator) to using local simulators.
+---
+
+# Migrate to local simulators
+
+In quantum computing, the choice between using simulators and quantum hardware is crucial for making progress in the field. While simulators are useful for testing and debugging, in this era of quantum utility, quantum development and industry advancement requires actual hardware. As part of the move to quantum utility, IBM Quantum&trade; will retire its cloud simulators on 15 May 2024. This guide explains in more detail why this is happening, and how to change from cloud to local simulators. 
```

Comment:
I think in the intro it would be worth emphasising that the "cloud simulators" are the ones people access through runtime, and maybe even listing the simulator names would be good. I think internally we have become familiar with the term "cloud simulators" but I worry that external users won't know what we mean and might panic that all simulators are being removed